components/map/AvalancheForecastZonePolygon.tsx

```diff
@@ -89,31 +87,10 @@ export const AvalancheForecastZonePolygon: React.FunctionComponent<AvalancheFore
         colorFor(zone.danger_level).alpha(zone.fillOpacity).string(),
       ],
     });
-    return <AnimatedPolygon fillColor={fillColor} {...polygonProps} />;
+    return <AnimatedGeojson fillColor={fillColor} {...polygonProps} />;
   } else {
     const fillOpacity = renderFillColor ? zone.fillOpacity : 0;
     const fillColor = colorFor(zone.danger_level).alpha(fillOpacity).string();
-    return <Polygon fillColor={fillColor} {...polygonProps} />;
+    return <Geojson fillColor={fillColor} {...polygonProps} />;
   }
 };
-
-// need to update to react-native-maps 1.6 to get rid of this
```

Comment:
Note to future self: use `Parameters` and `React.ComponentProps` to extract types from libraries that don't export things.